advanced-javascript/tests/arrays.test.js

```diff
@@ -18,43 +18,43 @@ describe('Arrays', () => {
   describe('`each`', () => {
     it('should be a function', () => {
       const each = arrayFunctions.each;
```

Comment:
You can use destructuring in instances like these:

`const { each } = arrayFunctions;`